Dockerfile

```diff
@@ -4,6 +4,7 @@ LABEL maintainer="info@redpencil.io"
 
 WORKDIR /app
 COPY package.json package-lock.json ./
+COPY public ./public/
```

Comment:
why was this necessary?

Reply:
because new npm now runs prepare scripts on `npm ci` :) which in our case includes the `svg-symbols` invocation, which needs the public folder to exist